### PR TITLE
Fix CMPopinCard style

### PR DESCRIPTION
**Pull request by antonia2929**
Trello ticket : 

**Detailed purpose of the PR**
- Add unit test for handleClose function
- Update popin width styles
Capture d’écran 2023-10-09 à 16 41 57

- Add closePopin function to close the popin when a lick is outside the component
Kapture 2023-10-09 at 16 44 06

**Result and observation**

**Testing Strategy**
- [ ] Already covered by tests
- [x] Manual testing
- [x] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/molecule/cm-popin/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+// import React from 'react';
```
Comment (EmelineLeduc) [inline packages/@coorpacademy-components/src/molecule/cm-popin/index.js:1]: Line 1 should be deleted

File: packages/@coorpacademy-components/src/molecule/cm-popin/index.js [hunk 2]
```diff
@@ -28,6 +30,8 @@ const IconType = {
     </div>
   )
 };
+// const preventDefault = e => e.preventDefault();
+// const stopPropagation = e => e.stopPropagation();
```
Comment (EmelineLeduc) [inline packages/@coorpacademy-components/src/molecule/cm-popin/index.js:34]: Clean code (line 33-34)

File: packages/@coorpacademy-components/src/molecule/cm-popin/index.js [hunk 3]
```diff
@@ -161,25 +179,31 @@ const CMPopin = props => {
 
     return null;
   };
-
+  // const handleCloseButton = useCallback(
```
Comment (EmelineLeduc) [inline packages/@coorpacademy-components/src/molecule/cm-popin/index.js:182]: Clean code

File: packages/@coorpacademy-components/src/molecule/cm-popin/test/on-close.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import test from 'ava';
```
Comment (EmelineLeduc) [inline packages/@coorpacademy-components/src/molecule/cm-popin/test/on-close.js:1]: Could you migrate the code to TS ?

File: packages/@coorpacademy-components/src/molecule/select-multiple/index.js [hunk 5]
```diff
@@ -96,6 +96,8 @@ const SelectMultiple = (
 
   const closeHandle = useCallback(e => {
     if (nodeRef && nodeRef.current && !nodeRef.current.contains(e.target)) {
+      // eslint-disable-next-line no-console
+      console.log('closeHandle');
```
Comment (EmelineLeduc) [inline packages/@coorpacademy-components/src/molecule/select-multiple/index.js:100]: Clean code : delete the console.log

File: packages/@coorpacademy-components/src/organism/mooc-header/index.js [hunk 6]
```diff
@@ -170,6 +170,8 @@ class MoocHeader extends React.Component {
     if (isSettingsOpen) {
       const menu = this.menuSettings;
       if (menu && !menu.contains(clickEvent.target)) {
+        // eslint-disable-next-line no-console
```
Comment (EmelineLeduc) [inline packages/@coorpacademy-components/src/organism/mooc-header/index.js:173]: Clean code : delete the console.log

File: packages/@coorpacademy-components/src/template/common/dashboard/style.css [hunk 7]
```diff
@@ -11,6 +11,10 @@
   height: 350px;
   width: 100%;
 }
+/*
```
Comment (EmelineLeduc) [inline packages/@coorpacademy-components/src/template/common/dashboard/style.css:14]: clean code

File: packages/@coorpacademy-components/src/template/common/search-page/style.css [hunk 8]
```diff
@@ -17,6 +17,10 @@
   background: white;
 }
 
+/* .hidden {
```
Comment (EmelineLeduc) [inline packages/@coorpacademy-components/src/template/common/search-page/style.css:20]: clean code

**Review by EmelineLeduc — CHANGES_REQUESTED** [review]

**Review by EmelineLeduc — APPROVED** [review]

**Review by silouone — APPROVED** [review]

File: packages/@coorpacademy-components/src/molecule/cm-popin/style.css [hunk 9]
```diff
@@ -151,6 +156,15 @@
...
 
+.popinItems .headerContent {
+  padding: 24px;
+  width: 100%;
+}
+
+.popinItems .popinHeader button {
+  margin-top: 12px;
+  margin-right: 12px;
+}
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/cm-popin/style.css:167]: je préfère avoir un className et par de tag générique (ici `button`). Risque d'effet de bord.

**Review by esa-coorp — CHANGES_REQUESTED** [review]

**Review by esa-coorp — APPROVED** [review]

**Review by silouone — APPROVED** [review]

**Pull request merged**